ocs_ci/helpers/odf_cli.py

```
@@ -186,24 +186,35 @@ def add_cli_to_path(self):
 
 class ODFCliRunner:
     def __init__(self) -> None:
-        self.binary_name = "odf"
+        self.binary_name = os.path.join(config.RUN["bin_dir"], "odf")
 
-    def run_command(self, command_args: Union[str, list]) -> str:
+    def run_command(self, command_args: Union[str, list], timeout=600) -> str:
```

Ruff (0.16.0)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)